server/data/authClientBuilder.ts

```diff
@@ -120,6 +122,10 @@ export class AuthClient {
     return body
   }
 
+  async getUsers(usernames: string[]): Promise<UserResult[]> {
+    return Promise.all(usernames.map(username => this.getUser(username)))
+  }
```

Comment:
could potentially add a bulk endpoint but numbers of users to retrieve isn't likely to be very large